packages/blocks/src/store/utils.js

```diff
@@ -19,6 +19,14 @@ export const getValueFromObjectPath = ( object, path, defaultValue ) => {
 	return value ?? defaultValue;
 };
 
+function isObject( candidate ) {
+	return (
+		typeof candidate === 'object' &&
```

Comment:
We called it `item` in Interactivity. I like this nomenclature, maybe we should update to keep consistency in the codebase. Ping @DAreRodz @luisherranz

Reply:
I need more context. What exactly are we referring to as "item" in the Interactivity API? What is being referred to here?

Reply:
https://github.com/WordPress/gutenberg/blob/81cde31199886a188fc47b80c66042c77b4dacb4/packages/interactivity/src/store.ts#L18

In Interactivity API, we check that several "items" are a JSON object in the same file (sources, state, config, proxy handlers, etc). So we call the variable to check "item".

"Candidate" seems also a good general definition, and would be ok to use the same name in all functions that are just checking that an item is an object (there are a few of them across the workspace).

Reply:
Oh, ok. You can change that if you want 🙂